src/Features/CSharp/Portable/RemoveAsyncModifier/CSharpRemoveAsyncModifierCodeFixProvider.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.InitializeParameter;
+using Microsoft.CodeAnalysis.CSharp.MakeMethodSynchronous;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.RemoveAsyncModifier;
+using Roslyn.Utilities;
+using KnownTypes = Microsoft.CodeAnalysis.MakeMethodAsynchronous.AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes;
+
+namespace Microsoft.CodeAnalysis.CSharp.RemoveAsyncModifier
+{
```

Comment:
```suggestion
namespace Microsoft.CodeAnalysis.CSharp.RemoveAsyncModifier
{
    using KnownTypes = AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes;
```

Reply:
`Microsoft.CodeAnalysis.MakeMethodAsynchronous` doesn't appear anywhere else in the list of usings so this doesn't save anything.